src/features/callAssignments/models/CallAssignmentModel.ts

```diff
@@ -132,6 +132,18 @@ export default class CallAssignmentModel extends ModelBase {
     this._repo.removeCaller(this._orgId, this._id, callerId);
   }
 
+  setCallerAccess(expose_target_details: boolean) {
```

Comment:
I don't like how the exposed API (the name of the argument) uses `snake_case`. I think it should use camel case as everything else. I'm surprised it's even allowed by the linter!

Reply:
Actually, the name of the function is not ideal either, IMO. What does "caller access" mean? Maybe it can be something like `setTargetDetailsExposed`, and the other one can be `setCallerNotesEnabled`?

Reply:
hehehe maybe i disabled a linting rule 😈